.editorconfig

```diff
@@ -0,0 +1,14 @@
+root = true
+
+[*]
+indent_style = space
+indent_size = 4
+charset = utf-8
+trim_trailing_whitespace = true
+insert_final_newline = true
+
+[Makefile]
+indent_style = tab
```

Comment:
We don't have a makefile here, let's drop this section please.

Reply:
There is a file in the docs folder.

Reply:
@jairhenrique I see :+1: